Sources/SwiftSyntaxBuilder/DeclBuildables.swift

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+extension StructDecl {
+  public init(_ identifier: String,
+       @MemberDeclListBuilder membersBuilder: () -> MemberDeclList = { .empty }) {
+    self.init(structKeyword: Tokens.struct,
+              identifier: SyntaxFactory.makeIdentifier(identifier),
+              members: MemberDeclBlock(leftBrace: Tokens.leftBrace.withLeadingTrivia(.spaces(1)),
+                                       members: membersBuilder(),
+                                       rightBrace: Tokens.rightBrace))
+  }
+}
+
+extension VariableDecl {
+  public init(_ letOrVarKeyword: TokenSyntax,
```

Comment:
Should we also have one for a more simple variable? That takes a identifier, type and expr as parameter?